docs/src/config/boundaries.md

```diff
@@ -1,8 +1,5 @@
-```@raw html
 <!--- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. --->
```

Comment:
I think the old code was correct: it embedded the copyright as a HTML comment (so that one would still get a copyright notice if they scraped the page). If you move to markdown comments, that is lost. 

Also, I don't understand why, but this change pastes the comment directly to the page:
<img width="852" height="190" alt="image" src="https://github.com/user-attachments/assets/fd238b58-2445-4960-9d86-9fb692843351" />